client/app/js/controllers/receiver.js

```diff
@@ -7,7 +7,12 @@ GLClient.controller('ReceiverSidebarCtrl', ['$scope', '$location', function($sco
 
 GLClient.controller('ReceiverTipsCtrl', ['$scope',  '$http', '$route', '$location', '$uibModal', 'ReceiverTips',
   function($scope, $http, $route, $location, $uibModal, ReceiverTips) {
-  $scope.tips = ReceiverTips.query();
+  $scope.tips = ReceiverTips.query(function(tips) {
+      angular.forEach(tips ,function(tip) {
+          tip.export = '/rtip/' + tip.id + '/export';
+        });
+      return tips;
+  });
```

Comment:
i'm not certain of this; good that you tried experimenting with the promises callback but a simple ```href="{{'/rtip/' + tip.id + '/export'}}``` in the link would make it more simple and would require less overhead by the client; in fact on the ng-repeat implement automagically 1 for cycle, with your we hand in having 2 for cycle: https://github.com/globaleaks/GlobaLeaks/blob/master/client/app/views/receiver/tips.html#L176

what do you think about it?